packages/pytest-simcore/src/pytest_simcore/docker_registry.py

```diff
@@ -261,7 +261,7 @@ def jupyter_service(docker_registry: str, node_meta_schema: dict) -> dict[str, A
     )
 
 
-@pytest.fixture(scope="session", params=["2.0.4"])
+@pytest.fixture(scope="session", params=["2.0.7"])

```

Comment:
question: should this not go into upstream master?

Reply:
no we don't use it there for anything. It's OK like this, since this is used for the CI.